debug.log

```diff
@@ -1 +1,19 @@
 [0927/232620.798:ERROR:crash_report_database_win.cc(428)] unexpected header
+[1003/221825.848:ERROR:crash_report_database_win.cc(428)] unexpected header
```

Comment:
remember to add *.log to your .gitignore file to make sure this sort of thing does not get committed